cores/arduino/cortex_handlers.c

```diff
@@ -176,3 +175,16 @@ void SysTick_Handler(void)
     return;
   SysTick_DefaultHandler();
 }
+
+static void (*usb_isr)(void);
+
+void USB_Handler(void)
+{
+  if (usb_isr)
```

Comment:
why not putting the vectors in RAM instead of using this trick?
SCB->VTOR=new vectors address